src/bech32.cpp

```diff
@@ -404,7 +398,8 @@ DecodeResult Decode(const std::string& str) {
 /** Find index of an incorrect character in a Bech32 string. */
 std::string LocateErrors(const std::string& str, std::vector<int>& error_locations) {
     if (str.size() > 90) {
```

Comment:
In commit "Use std::iota instead of manually pushing range" (2fa4fd196176160a5ad0a25da173ff93252b8103)

It would be good to add `error_locations.clear()` to the top of this function, if the intent is now to overwrite `error_locations` instead of appending to it. Before this commit, the function consistently appended without overwriting, and now it sometimes overwrites, sometimes appends, which is less consistent.

Reply:
Done in new commit to avoid invalidating the two ACKs.

Reply:
Sorry to nit the nit, if the intent is to always return something new instead of modifying in place, why not return both values in the return value with e.g. `std::pair` or `std::tuple` instead?

Reply:
@laanwj no worries, I agree. Addressed your nit-nit :)